mass_mailing_keep_archives/mass_mailing.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
```

Comment:
have ome file per model in /models: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#directories